tests/context-menu.spec.ts

```diff
@@ -1,4 +1,262 @@
-import { test, expect } from '@playwright/test';
+import { test, expect, Page } from '@playwright/test';
+import {
+  makeTableCommand,
+  pasteInMonaco,
+  TableTypes,
+  waitForLoadingDone,
+  generateVarName,
+} from './utils';
+
+test.describe.configure({ mode: 'serial' });
+
+async function openAdvancedFilters(page: Page) {
+  await page
+    .locator('.iris-grid .grid-wrapper')
+    .click({ button: 'right', position: { x: 20, y: 20 } });
+  await page.getByRole('button', { name: 'Advanced Filters' }).click();
+}
```

Comment:
This method does open advanced filters, but it just seems to be doing just the first column of the first grid (e.g. you can't pass in a specific column or even a specific grid to open the advanced filter on). I'd rather be more specific with the name of the function (e.g. `openFirstAdvancedFilters`), or allow passing in the grid locator and an optional offset, e.g.:
```
/**
 * Open advanced filters for a grid.
 * @param gridLocator Locator to specify which grid to open the advanced filters on
 * @param xOffset Offset of the column to open advanced filters for. Defaults to the first column.
 */
async function openAdvancedFilters(gridLocator: Locator, xOffset = 20) 
  ...
}
```